examples/struct_indices/SQLIndexDemo-Context.ipynb

```diff
@@ -203,31 +203,24 @@
    "outputs": [],
    "source": [
     "# GPT-extracted context\n",
-    "sql_context_builder = SQLContextBuilder(sql_database)\n",
     "city_stats_text = (\n",
```

Comment:
Is this generated by GPT? If not can we use GPT to automatically generate this text given the table schema?

Reply:
no this is supposed to represent the raw document! gpt will extract context from this document.

i know it's silly since the "document" here is a sentence. it's a toy example

Reply:
It's more just use case discovery at this point, but the high-level is if the user didn't want to manually set a summary/text description of a table but wanted to infer it from a collection of documents